exercises/series/src/Example.hs

```diff
@@ -4,5 +4,6 @@ import Data.Char (digitToInt)
 import Data.List (tails)
 
 slices :: Int -> String -> [[Int]]
+slices 0 _ = [[]]
```

Comment:
OK, in that case my recommendation is to merge this as-is. Though really, I would not object to not having it!

Maybe when the day comes when someone creates `series/canonical-data.json` they will decide explicitly to exclude 0... well we can revisit then.